pkg/ring/kv/memberlist/broadcast.go

```diff
@@ -0,0 +1,65 @@
+package memberlist
+
+import (
+	"fmt"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/hashicorp/memberlist"
+)
+
+type ringBroadcast struct {
+	logger   log.Logger
+	key      string
+	content  []string // Description of what is stored in this value. Used for invalidation.
+	version  uint     // local version of the value, generated by merging this change
+	msg      []byte   // encoded key and value
+	finished func(b ringBroadcast)
+}
+
```

Comment:
Maybe add a comment that it implements `memberlist.Broadcast`. Was wondering where this was being called.

Reply:
That's a good idea, will do.